eng/pipelines/templates/steps/build-test.yml

```
@@ -20,49 +20,53 @@ steps:
     env:
       GO111MODULE: 'on'
 
-  - ${{ if eq(parameters.RunTests, 'true') }}:
-    - pwsh: |
-        go get github.com/jstemmer/go-junit-report
-        go get github.com/axw/gocov/gocov
-        go get github.com/AlekSi/gocov-xml
-        go get github.com/matm/gocov-html
-        go get -u github.com/wadey/gocovmerge
-      displayName: "Install Coverage and Junit Dependencies"
-      workingDirectory: '${{parameters.GoWorkspace}}'
+  - pwsh: ../eng/scripts/run_tests.ps1
```

I would make this an absolute path.

```suggestion
  - pwsh: /eng/scripts/run_tests.ps1
```

---

I added this change but it threw an error in DevOps [here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1021652&view=logs&j=35205994-0183-5e87-7c1e-dd3d552c3593&t=8ccf51ca-6d04-5845-9c4d-7ec54a8f0a12&l=21).

---

Oh. I recommend invoking this via the [full powershell task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/powershell?view=azure-devops) rather than calling the script via an inline `pwsh` command.